src-docs/src/views/badge/badge_example.js

```diff
@@ -261,5 +277,29 @@ export const BadgeExample = {
       snippet: notificationBadgeSnippet,
       demo: <NotificationBadge />,
     },
+    {
+      title: 'Badge Group',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: badgeGroupSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: badgeGroupHtml,
+        },
+      ],
+      text: <p>Used to group Badges together </p>,
```

Comment:
This will need to be fleshed out and watch your casing.
```suggestion
      text: <p>Grouping badges with EuiBadgeGroup, ensures they wrap and truncate properly.</p>,
```